### PR TITLE
fix React scss ident

### LINKED ISSUE #7446: [React] Inconsistence between .editorconfig and some scss files
##### **Overview of the issue**

For React project, some `scss` files have parts with strange indentation, not according to `.editorconfig`.

##### **Motivation for or Use Case**

It's not very cool cause of inconsistency in the style.

.editorconfig is:
```
[*.{ts,tsx,js,jsx,json,scss,sql,ejs}]
indent_style = space
indent_size = 2
```

Files found are at least 

```
./src/main/webapp/app/shared/layout/password/password-strength-bar.scss
./src/main/webapp/app/shared/layout/header/header.scss
./src/main/webapp/app/modules/home/home.scss
./src/main/webapp/app/app.scss
```

Founded thanks command `$ find -name "*ss" ! -path "*node_modules*" -print0| xargs -0i grep -El '^\s{4}' {}`, then verified manually. 

##### **Related issues**
Nothing found

##### **JHipster configuration**
* All as default except:
  * react
  * scss

##### **Suggest a Fix**
Make a PR that correct this

##### **JHipster Version(s)**
JHipster 5.0.0-beta.0

- [ X] Checking this box is mandatory (this is just to show you read everything)

### PR DESCRIPTION
**Pull request by agaspardcilia**
fix #7446
- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [x] Tests are added where necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/react/src/main/webapp/app/shared/layout/header/header.scss.ejs [hunk 1]
```diff
@@ -16,80 +16,80 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -%>
-$header-color:                  #fff;
+header-color:                  #fff;
```
Comment (deepu105) [inline generators/client/templates/react/src/main/webapp/app/shared/layout/header/header.scss.ejs]: this is wrong, vars should start with $

**Comment by JulioJu** [comment]
@agaspardcilia there is other files with 4 indents.

**Comment by JulioJu** [comment]
Again, I've worked during an other contributor push his PR, as there isn't comment I believed it was unecessary to say I work on it… ;-)

**Comment by JulioJu** [comment]
@agaspardcilia 
Ok I work again on it ;-)… I work with js-beautify as I've proposed in #7438 to lint scss files ;-)… No cumbersome complexity ;-)…

**Comment by JulioJu** [comment]
@agaspardcilia sorry. I must say when I work on a thing ;-)… Sorry.

**Comment by agaspardcilia** [comment]
@JulioJu don't worry, it's was like 15 minutes of wark at best. :relaxed: Besides, I haven't saw #7438 yesterday, I thought #7446 was coming from nowhere. :sweat_smile:

**Comment by deepu105** [comment]
@JulioJu Prettier supports Sass and css we just need to configure a transform stream for it

**Comment by JulioJu** [comment]
@deepu105 so cool ! I've didn't see that ! Just for information, I've finished some tests to see if js-beautify format well the code for scss with the correct indentation… Seems to work fine ! But probably prettier is better… Much more contributors than js-beautify ;-)…

**Comment by JulioJu** [comment]
@agaspardcilia 
Sorry, I'm beginner… Maybe it will be my first PR merged ;-)… 
As I'm afraid to commeet errors I'm very slow… I've hesitated to make a PR directly, or say I work on it, but as I was not sure…

**Pull request closed without merging**